service/routes.py

```diff
@@ -182,38 +217,11 @@ def delete(self, inventory_item_id):
             app.logger.info(
                 "Inventory item with id [%s] was deleted", inventory_item_id
             )
+            return "", status.HTTP_204_NO_CONTENT
+        # a 404 is returned if a DELETE is sent for a nonexistant item
+        return "", status.HTTP_404_NOT_FOUND
```

Comment:
I made this modification because I saw a test case in the test_service.py:
def test_delete_unknown_item(self):
        """Ensure a 404 is returned if a DELETE is sent for a nonexistant item"""

Or maybe we should change this test case?